tsconfig.json

```
@@ -1,48 +1,38 @@
+// Aggregate project referencing all composite sub-projects, for use with `tsc`
+// type checker and `ts-node`.
+// NOTE: Only refer to this file via the symlink in the root, otherwise it will
```

```suggestion
// NOTE: Only refer to this file via `tsconfig.json` in the root, otherwise it will
```
The comment is now outdated and should either be fixed or removed.